src/renderer/atlas/CustomGlyphs.cpp

```diff
@@ -0,0 +1,1265 @@
+    /* Pos_3_4         */ { 3.0f / 4.0f, 0.0f },
+
+    /* Pos_2_6         */ { 2.0f / 6.0f, 0.0f },
+    /* Pos_3_6         */ { 3.0f / 6.0f, 0.0f },
+    /* Pos_5_6         */ { 5.0f / 6.0f, 0.0f },
+
+    /* Pos_1_8         */ { 1.0f / 8.0f, 0.0f },
+    /* Pos_3_8         */ { 3.0f / 8.0f, 0.0f },
+    /* Pos_5_8         */ { 5.0f / 8.0f, 0.0f },
+    /* Pos_7_8         */ { 7.0f / 8.0f, 0.0f },
+
+    /* Pos_2_9         */ { 2.0f / 9.0f, 0.0f },
+    /* Pos_3_9         */ { 3.0f / 9.0f, 0.0f },
+    /* Pos_5_9         */ { 5.0f / 9.0f, 0.0f },
+    /* Pos_6_9         */ { 6.0f / 9.0f, 0.0f },
+    /* Pos_8_9         */ { 8.0f / 9.0f, 0.0f },
+
+    /* Pos_2_12        */ { 2.0f / 12.0f, 0.0f },
+    /* Pos_3_12        */ { 3.0f / 12.0f, 0.0f },
+    /* Pos_5_12        */ { 5.0f / 12.0f, 0.0f },
+    /* Pos_6_12        */ { 6.0f / 12.0f, 0.0f },
+    /* Pos_8_12        */ { 8.0f / 12.0f, 0.0f },
+    /* Pos_9_12        */ { 9.0f / 12.0f, 0.0f },
+    /* Pos_11_12       */ { 11.0f / 12.0f, 0.0f },
+};
+
+static constexpr char32_t BoxDrawing_FirstChar = 0x2500;
+static constexpr u32 BoxDrawing_CharCount = 0xA0;
+static constexpr Instruction BoxDrawing[BoxDrawing_CharCount][InstructionsPerGlyph] = {
+    // U+2500 ─ BOX DRAWINGS LIGHT HORIZONTAL
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_1_2, Pos_1_1, Pos_1_2 },
+    },
+    // U+2501 ━ BOX DRAWINGS HEAVY HORIZONTAL
+    {
+        Instruction{ Shape_HeavyLine, Pos_0_1, Pos_1_2, Pos_1_1, Pos_1_2 },
+    },
+    // U+2502 │ BOX DRAWINGS LIGHT VERTICAL
+    {
+        Instruction{ Shape_LightLine, Pos_1_2, Pos_0_1, Pos_1_2, Pos_1_1 },
+    },
+    // U+2503 ┃ BOX DRAWINGS HEAVY VERTICAL
+    {
+        Instruction{ Shape_HeavyLine, Pos_1_2, Pos_0_1, Pos_1_2, Pos_1_1 },
+    },
+    // U+2504 ┄ BOX DRAWINGS LIGHT TRIPLE DASH HORIZONTAL
+    {
+        Instruction{ Shape_LightLine, Pos_0_1, Pos_1_2, Pos_2_9, Pos_1_2 },
+        Instruction{ Shape_LightLine, Pos_3_9, Pos_1_2, Pos_5_9, Pos_1_2 },
+        Instruction{ Shape_LightLine, Pos_6_9, Pos_1_2, Pos_8_9, Pos_1_2 },
+    },
+    // U+2505 ┅ BOX DRAWINGS HEAVY TRIPLE DASH HORIZONTAL
+    {
+        Instruction{ Shape_HeavyLine, Pos_0_1, Pos_1_2, Pos_2_9, Pos_1_2 },
+        Instruction{ Shape_HeavyLine, Pos_3_9, Pos_1_2, Pos_5_9, Pos_1_2 },
+        Instruction{ Shape_HeavyLine, Pos_6_9, Pos_1_2, Pos_8_9, Pos_1_2 },
+    },
+    // U+2506 ┆ BOX DRAWINGS LIGHT TRIPLE DASH VERTICAL
+    {
+        Instruction{ Shape_LightLine, Pos_1_2, Pos_0_1, Pos_1_2, Pos_2_9 },
+        Instruction{ Shape_LightLine, Pos_1_2, Pos_3_9, Pos_1_2, Pos_5_9 },
+        Instruction{ Shape_LightLine, Pos_1_2, Pos_6_9, Pos_1_2, Pos_8_9 },
+    },
+    // U+2507 ┇ BOX DRAWINGS HEAVY TRIPLE DASH VERTICAL
+    {
+        Instruction{ Shape_HeavyLine, Pos_1_2, Pos_0_1, Pos_1_2, Pos_2_9 },
+        Instruction{ Shape_HeavyLine, Pos_1_2, Pos_3_9, Pos_1_2, Pos_5_9 },
+        Instruction{ Shape_HeavyLine, Pos_1_2, Pos_6_9, Pos_1_2, Pos_8_9 },
+    },
```

Comment:
*Just a note*

These vertical dashed lines, when drawn one below the other, don't have gaps when line-height is 1. The top and bottom bars *connect* to the top/bottom bar of the adjacent vertical dashed line. 

![image](https://github.com/microsoft/terminal/assets/55626797/e50bbf35-154f-4f48-807e-6dd6d06c7bed)


*we can (if we want to) follow up on issues around individual glyph shape(forms ?) later 😄*

Reply:
Honestly, I would call those "font bugs that we are trying to get away from" :D

Reply:
> [...] and IMO the custom rendered one looks better.

I'm glad to hear that! I think that the vertical ┇ lines are intended to have consistent gaps no matter the line height. The approach in this PR is not perfect, because it gets into rounding errors at smaller font sizes, but I hope it'll be good enough for an initial version.